<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogDelete</class>
 <widget class="QDialog" name="DialogDelete">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>436</width>
    <height>171</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>删除操作选项</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTableWidget" name="tableWidget"/>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_delete_absolutely">
       <property name="text">
        <string>彻底删除</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_delete_recyclebin">
       <property name="text">
        <string>到回收站</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_delete_onlytable">
       <property name="text">
        <string>移除列表</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_delete_canceled">
       <property name="text">
        <string>取消操作</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
